### DISCUSSION TITLE
What would you like "TLSGrab" option to output?

### DISCUSSION THREAD

**Discussion by CyberStryk** [projectdiscovery/httpx#1471] [Ideas]
What output would you like to see (if any) when the -tls-grab httpx option is provided?

I've been working on some new features related to vhosting and TLS Certificates including PR that I believe addresses an issue with the TLSProbe option not working when providing a list of targets. 

For my purposes I found it useful to show some of the TLSData in the httpx output, but I wanted to start a discussion on what people would like to see. 

For my purposes the most useful information was the SubjectCN and SubjectAN which can be used to determine, a host's virtual hosts and other targets of interest, etc.

Here's what I'm currently which is just a list of names pulled from the SubjectCN and SubjectANs:
└─$ httpx -tls-grab -u https://127.0.0.1

![image](https://github.com/projectdiscovery/httpx/assets/15826683/64976ba5-2b2d-4773-b3ee-96430d18e765)

What are your thoughts? Is this useful output? What would you like to see added or changed?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
